src/helpers/constants/error.messages.js

```diff
@@ -49,5 +49,6 @@ export default {
   parentCommentId: 'You can not reply to a reply',
   notCommentOwner: 'You must be the owner of the comment in order to proceed',
   noBookmark: 'You have no bookmarks',
-  noSlug: 'There is no article with the slug provided'
+  noSlug: 'There is no article with the slug provided',
+  noHighlight: 'No highlited text for this article',
```

Comment:
@alainburindi it's `highlighted` not `highlited`

This mistake is happening across several files so please correct in other places, as well.